proto/private/generate_sums.sh

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/bash
-
```

Comment:
nit: I see why you might want to remove the shebang line here as this hardcodes the path to bash to the location on macos, but I don't think this is right. For one thing you've left the file as executable, so if the sheband is removed the executable bit should be removed as well. I don't have a mac but afaik there's a symlink from /bin/bash on macos, or worst case you could do something like `#!/usr/bin/env bash`

Reply:
@Yannic could you add it back?

Reply:
`/usr/bin/bash` definitely doesn't exist on macOS (10.14 & 10.15), so leaving it as it is makes the script unusable for me. Replaced it with `#!/usr/bin/env bash`.